contracts/src/v0.7/ArbitrumValidator.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0; // Could we use 0.8.0
+
+import "./interfaces/ArbitrumInboxInterface.sol";
+import "./interfaces/AggregatorValidatorInterface.sol";
+import "../v0.6/SimpleWriteAccessController.sol";
+
+contract ArbitrumValidator is SimpleWriteAccessController, AggregatorValidatorInterface {
+  address s_arbitrumInbox;
+  address s_flags;
+  // Follows: https://eips.ethereum.org/EIPS/eip-1967
+  address s_arbitrumFlag = address(bytes20(bytes32(uint256(keccak256("chainlink.flags.arbitrum-offline")) - 1)));
+  address s_gasConfigAccessController;
+  address s_refundableAddress;
+
+  struct GasConfiguration {
+    uint32 maximumGasPrice;
+    uint256 gasCostL2;
+  }
+  GasConfiguration internal s_gasConfig;
+
+  /**
+   * @param inboxAddress address of the Arbitrum Inbox L1 contract
+   * @param flagAddress address of the Chainlink L2 Flags contract
+   * @param gasConfigAccessController address of the access controller for managing gas price on Arbitrum
+   * @param maximumGasPrice maximum gas price to pay on L2
+   * @param gasCostL2 value to send to L2 to cover gas fee
+   * @param refundableAddress address where gas excess on L2 will be sent
+   */
+  constructor(
+    address inboxAddress, 
+    address flagAddress,
+    address gasConfigAccessController,
+    uint32 maximumGasPrice,
+    uint256 gasCostL2,
+    address refundableAddress
+  ) 
+    public
+  {
+    s_arbitrumInbox = inboxAddress;
+    s_flags = flagAddress;
+    s_gasConfigAccessController = gasConfigAccessController;
+    s_refundableAddress = refundableAddress;
+    // TODO: Is it possible to give default access to the aggregator?
+    // addAccess(aggregatorAddress);
+    _setGasConfigurationInternal(maximumGasPrice, gasCostL2);
+  }
+  
+  // Accept ETH funds
+  fallback() external payable {}
```

Comment:
Let's add a withdraw for ETH funds as well, only owner.

Reply:
Added a couple of them. Please check if they are correct